src/utils/urlJoin.ts

```diff
@@ -0,0 +1,6 @@
+export default function urlJoin(...parts: Array<any>) {
```

Comment:
Had to `any` this. Could not get the types to be valid with `Array<string | null | undefined>`

Reply:
seems to work for me https://www.typescriptlang.org/play/?ssl=1&ssc=1&pln=7&pc=1#code/KYDwDg9gTgLgBAE2AMwIYFcA29noHYDGMAlhHnOlJgFITF4AUAdC2KrAM4BccAglFFQBPADwcYUegHM4AHzh4smORTxJk9YAgB8ASjgBvALAAoOObhRgMSuTacmVhOgLAGDKABoFoGPoC82oamFqFwBGTicBzAUgC2wHjw-j4gySmKmMoA-HAA5HlwPADKEtIMeL66jsBgmKiuDAD0AHoAOk0A1E3eBboA3CFh5lY2UOQx8YnwuQAGACQGUACENXUNbk0dnfM9+Xm6AL5Ni5MJSYezRZaDZnCHvQe3h6ZAA

Reply:
My latest PR updates typescript. Maybe that fixed it? If no errors then let’s do that